tests/urls.py

```diff
@@ -9,4 +14,6 @@
     url(r'^api/', include('osmaxx.excerptexport.rest_api.urls', namespace='excerptexport_api')),
     url(r'^job_progress/', include('osmaxx.job_progress.urls', namespace='job_progress')),
     url(r'^pages/', include('osmaxx.core.urls', namespace='pages')),
+    # we're serving through nginx in production, but we need this here so we can test more easily!
+    url(r'^%s(?P<path>.*)$' % re.escape(settings.MEDIA_URL.lstrip('/')), serve, kwargs={'document_root': settings.MEDIA_ROOT}),
```

Comment:
Oh, because it isn't there ;-)

It is here:
https://github.com/django/django/blob/master/django/conf/urls/static.py